openedx/features/student_account/helpers.py

```diff
@@ -71,6 +81,15 @@ def save_user_utm_info(req, user):
     :return:
     """
     def extract_param_value(request, param_name):
+        """
+        Get value of params from request
+
+        Arguments:
+            request (Request): Request Object
```

Comment:
Please add missing para `param_name` in dosctring

**Note: Please run pylint again, this and similar issues will be highlighted in pylint report**